test/assets/combination/testdata.go

```diff
@@ -1,12 +1,12 @@
-package combinationTestData
+package testdata
 
-var CombinationInput map[string][]string = map[string][]string{
+var CombinationInput = map[string][]string{
```

Comment:
This test data is relatively small, do we really need to keep it in separate files? can we just define it up front, as literals, with the test cases themselves?

I'm usually all for DRY, but the one exception is test cases where:
- I don't want to _hide_ my expectations from the reader
- I don't want to create a shared dependency between tests/testcases; i.e. tweaking shared testdata usually always has unintended consequences, so we almost always end up writing functions to mutate that data -- invariably this leads to:
  - test cases being hard to grok; e.g. `input: withFoo(withBar(withBaz(mySharedInput), baz), bar), foo),`
  - more maintenance